src/umpire/strategy/AllocationAdvisor.cpp

```diff
@@ -66,11 +66,10 @@ AllocationAdvisor::AllocationAdvisor(
 void* AllocationAdvisor::allocate(size_t bytes)
 {
   void* ptr = m_allocator->allocate(bytes);
-  auto alloc_record = new util::AllocationRecord{ptr, bytes, this->shared_from_this()};
 
   m_advice_operation->apply(
       ptr,
-      alloc_record,
+      nullptr,
```

Comment:
I don't understand what is happening here.  What was this change for?

Reply:
It used to create a new AllocationRecord that was never used, so now I skip that step.